dist/10-10EZ-schema.json

```diff
@@ -2211,6 +2211,67 @@
     "campLejeune": {
       "type": "boolean"
     },
+    "hasTeraResponse": {
+      "type": "boolean"
+    },
+    "didRadiationCleanup": {
+      "type": "boolean"
+    },
+    "gulfWarService": {
+      "type": "boolean"
+    },
+    "gulfWarStartDate": {
+      "$ref": "#/definitions/date"
+    },
+    "gulfWarEndDate": {
+      "$ref": "#/definitions/date"
+    },
+    "didCombatOperations": {
+      "type": "boolean"
+    },
+    "exposedToAgentOrange": {
+      "type": "boolean"
+    },
+    "agentOrangeStartDate": {
+      "$ref": "#/definitions/date"
+    },
+    "agentOrangeEndDate": {
+      "$ref": "#/definitions/date"
+    },
+    "exposedToAirPollutants": {
+      "type": "boolean"
+    },
+    "exposedToAsbestos": {
+      "type": "boolean"
+    },
+    "exposedToChemicals": {
+      "type": "boolean"
+    },
+    "exposedToMustardGas": {
+      "type": "boolean"
+    },
+    "exposedToOccupationalHazards": {
+      "type": "boolean"
+    },
+    "exposedToShad": {
```

Comment:
no exposure to shad in their schema

Reply:
Will note this to UX and ask how to proceed.